seqr/utils/search/elasticsearch/es_utils_tests.py

```diff
@@ -138,7 +138,12 @@ def call_request_json(self, index=-1):
               'consequence_terms': [
                   'frameshift_variant'
               ]
-            }
+            },
+            {
```

Comment:
it would be better/ more accurate to add this to the mock SV variant, instead of here